backend/lib/endpoints/users.js

```diff
@@ -1,36 +1,34 @@
 const httpErrors = require("http-errors");
 
+const { getUserByIdSchema } = require("./schema/posts");
```

Comment:
Is the path here correct? I think this is supposed to require `./schema/users` since that is where `getUserbyIdSchema` is located, right?

Reply:
No this was a bug, well spotted, fixed now. Actually strange that I didn't notice the name `import` didn't exist. One of these examples where TypeScript would definitely help.

Reply:
Yes, I'm actually a big fan of TypeScript, although it does become a pain to have to rebuild to transpile from TypeScript to JavaScript. We could potentially set up a gulp script combined with nodemon to automatically rebuild on changes though.